docs/01 - Contributing/README.md

```diff
@@ -17,6 +17,10 @@
 Check out the [Stellar Contribution Guide](https://github.com/stellar/.github/blob/master/CONTRIBUTING.md) that apply to
 all Stellar projects.
 
+## Quick Start Anchor Platform
```

Comment:
We also have quick start instructions in the main README https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/docs/README.md#quickstart. Should we merge the two?

Reply:
Removed and left one-line fix.